knora/dsplib/schemas/lists-only.json

```diff
@@ -98,38 +98,39 @@
     },
     "type": "object",
     "properties": {
-        "name": {
```

Comment:
Until now, the single list items (=root node with its children) were validated, not the "lists" section as a whole. For the different use cases inside dsp-tools, it makes more sense to do it this way.